treeherder/webapp/schemas/artifacts.py

```diff
@@ -0,0 +1,55 @@
+import jsonschema
+
+from rest_framework import viewsets
+from rest_framework.response import Response
+from rest_framework.decorators import list_route
+
+from treeherder.etl.schema import IngestionDataSchema
+
+
+class ArtifactViewSet(viewsets.ViewSet):
+
+    """
+    ViewSet for validating some possible Job artifacts against a schema
+
+    """
+
+    @list_route()
+    def definition(self, request):
```

Comment:
I would change the endpoint name to 'schema'. [The rest framework documentation](http://www.django-rest-framework.org/api-guide/metadata/) suggests to use a custom metadata class to do that